src/components/FilmList/FilmList.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Films from './Films';
+import { isNonEmptyArray } from './deps';
+import { FilmLists } from './type';
+
+const films: any = ['How to train your dragon', 'Fast & Furious 6', 'Gladiator', 'Captain Marvel', 'Iron Man 3'];
+const resultArray: any = [];
+
+class FilmList extends React.Component {
+  state: FilmLists = {};
+  _isMounted = false;
+
+  componentDidMount() {
+    this._isMounted = true;
+    this._isMounted && films.forEach((props: any) => {

```

Comment:
Useless check: you know this._isMounted is true because you set it to true in the previous line.

Reply:
I have changed it. Please check now.